.github/workflows/zeebe-benchmark.yml

```diff
@@ -213,6 +213,16 @@ jobs:
     name: Deploy
     needs:
       - calculate-image-tag
+      - build-zeebe-image
+      - build-benchmark-images
+    # Deploy will be run:
+    # - after build jobs if they are not skipped
+    # - after calculate-image-tag if build jobs are skipped
+    # this is to have the possibility to re-deploy the benchmarks without rebuilding the images
```

Comment:
```
    # To have the possibility to re-deploy the benchmarks without rebuilding the images, the deploy will be run, when build is skipped or successful.
```

:wrench: The comment was a bit confusing to me. I would simply focus on the why only.

Reply:
Thank you for the suggestion, added your comment instead.